$NetBSD: patch-ag,v 1.21 2014/08/25 08:23:22 wiz Exp $

Use LDFLAGS from environment.
Similar fix already in current Advanced release.

--- buildtools/Makefile.orig	2010-10-28 07:01:24.000000000 +0000
+++ buildtools/Makefile
@@ -37,7 +37,7 @@ typegen.o endiangen.o:%.o:%.c
 	$(CC_FOR_BUILD) -c $(CFLAGS_FOR_BUILD) -o $@ $<
 
 $(PROGS):%:%.o
-	$(LD_FOR_BUILD) -o $@ $<
+	$(LD_FOR_BUILD) $(LDFLAGS) -o $@ $<
 
 distclean clean: cleanlocal
 .PHONY: cleanlocal
